Makefile

```diff
@@ -451,3 +451,8 @@ package-rpm-suse package-windows package-standalone-macosx \
 package-generic-unix: $(PACKAGES_SOURCE_DIST_FILE)
 	$(verbose) $(MAKE) -C packaging $@ \
 		SOURCE_DIST_FILE=$(abspath $(PACKAGES_SOURCE_DIST_FILE))
+
+generate-config:
+	rm -rf generated/rabbitmq.*.config
+	./cuttlefish -e . -s ./schema -c rabbitmq.conf.d/rabbitmq.conf -f rabbitmq
```

Comment:
I opened https://github.com/ninenines/erlang.mk/issues/503 to track this. Feel free to remind me if it's not done when it's time to merge, it shouldn't take too long to do.